src/components/portal/pages/reviews/Home.tsx

```diff
@@ -37,17 +65,29 @@ export default class ReviewsHome extends PortalPageBase {
     )
   }
   
-  renderLoadReview(proposalId) {
+  renderLoadReview(proposalId, reviewId) {
+    const review = ReviewsHome.allReviews[proposalId].reviews[reviewId];
+    console.log('loadReview', review);
     return (
       <div key="load">
         Loading Review for Proposal {proposalId}
-        <ReviewLoad proposalId={proposalId} {...this.props} />
+        <ReviewLoad review={review} {...this.props} />
       </div>
     );
   }
 
-  getProposalList() {
-    const filtered = this.state.allReviews.filter((item) => item.users.includes(this.state.curUser));
+  getReviewList() {
+    const filtered:any[] = [];
+    ReviewsHome.allReviews.forEach((proposal) => {
+      console.log('80, reviews/home', proposal);
+      if(proposal.reviewers !== 'complete') {
```

Comment:
Are there other states that these could be in? And if so, would it be a good idea to contain the strings as `var`s or `const`s somewhere locally (or even globally) to avoid weird typo errors or anything similar?

Reply:
Yes, all of these states are going to be database driven. Once we have everything set up we will be able to ditch the string matching and add functions like "isReviewComplete()" or "isReviewSubmitted()" to validate it against known DB states.